resources/views/frontend/collection/form.blade.php

```diff
@@ -0,0 +1,16 @@
+@php FormRenderer::setTheme($form, 'filter') @endphp
+
+<section class="filters">
+    <div class="container content-section">
+        <div class="expandable">
+            <div class="row">
+                <div class="col-md-4 col-xs-6 bottom-space">
```

Comment:
I was just thinking , whether 
```suggestion
                <div class="col-xs-6">
```
 wouldn't look better (now there is this "empty space" instead of the third filter). but let's discuss it once it's on the test visible to everyone. however. please - no `bottom-space` here

Reply:
but we have _bottom-space_ everywhere, also I'd keep stacking to the left, its similar when you change resolution on other pages, inputs ale left-aligned instead of centering

Reply:
ok. stacking to the left make sense. but bottom-space was here in catalog view, because there is another row of input fields... but not in this case. it's making unproportional space in the bottom. 
still we fine-tune these details in release branch. technically it's superb.